homeassistant/components/homewizard/number.py

```diff
@@ -0,0 +1,69 @@
+"""Creates HomeWizard Number entities."""
+from __future__ import annotations
+
+from homeassistant.components.number import NumberEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import PERCENTAGE
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import DOMAIN
+from .coordinator import HWEnergyDeviceUpdateCoordinator
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up numbers for device."""
+    coordinator: HWEnergyDeviceUpdateCoordinator = hass.data[DOMAIN][entry.entry_id]
+
+    if coordinator.data["state"]:
+        async_add_entities(
+            [
+                HWEnergyNumberEntity(coordinator, entry),
+            ]
+        )
+
+
+class HWEnergyNumberEntity(
+    CoordinatorEntity[HWEnergyDeviceUpdateCoordinator], NumberEntity
+):
+    """Representation switchable entity."""
```

Comment:
Stale comment

Reply:
Whoops, fixed in https://github.com/home-assistant/core/pull/82378/commits/76c23c1ddded45d9832c681ef31afa2d58554e11